java/src/main/native/src/dtype_utils.hpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+namespace dtype_utils{
```

Comment:
nit: This is really minor because all of the functions are inline and the header is private to the JNI code, but I would prefer to have the namespace have something with cudf and java/jni in it.

Reply:
Thanks @mythrocks , I have made the changes as per suggestion.